File: src/main/java/com/rdvdev2/timetravelmod/impl/ModBiomes.java
package com.rdvdev2.timetravelmod.impl;

import net.minecraft.util.registry.Registry;
import net.minecraft.util.registry.RegistryKey;
import net.minecraft.world.biome.Biome;

public class ModBiomes {

    public static final RegistryKey<Biome> OLD_WEST = RegistryKey.of(Registry.BIOME_KEY, Mod.identifier("old_west"));
}

File: src/main/kotlin/com/rdvdev2/timetravelmod/impl/ModBiomes.kt
package com.rdvdev2.timetravelmod.impl

import com.rdvdev2.timetravelmod.impl.Mod.identifier
import net.minecraft.util.registry.Registry
import net.minecraft.util.registry.RegistryKey
import net.minecraft.world.biome.Biome

object ModBiomes {

    @JvmField
    val OLD_WEST: RegistryKey<Biome> = RegistryKey.of(Registry.BIOME_KEY, identifier("old_west"))
}